web/apps/labelstudio/src/config/ApiConfig.example.js

```diff
@@ -0,0 +1,139 @@
+/* 
+to mock a request, you can define the endpoint with a schema that looks like this:
+endpointName:{                              //endpointName is what is used when calling callAPI, passed in as a string
+      method: "GET",                        //Optional. defaults to GET. options are GET, POST, PATCH, DELETE
+      mock: undefined,                      //Optional. function that returns the mock data
```

Comment:
also would love to see the signature of the function, it strictly expects some args like url, params